security-admin/src/main/resources/conf.dist/ranger-admin-site.xml

```diff
@@ -225,6 +225,48 @@
 		<value>Mozilla,chrome</value>
 	</property>
 	<!-- SSO Properties Ends-->
+	<!-- Altiscale SSO Properties Starts-->
+	<property>
+		<name>ranger.web.authentication.alt-kerberos.enabled</name>
+		<value>false</value>
+	</property>
+	<property>
+		<name>ranger.web.authentication.alt-kerberos.signature.secret.file</name>
+		<value>/etc/verticloud/secrets/auth_secret</value>
+	</property>
+	<property>
+		<name>ranger.web.authentication.alt-kerberos.token.validity</name>
+		<value>36000</value>
+	</property>
+	<property>
+		<name>ranger.web.authentication.alt-kerberos.cookie.domain</name>
+		<value>sn-zoran215.test.altiscale.com</value>
+	</property>
+	<property>
+		<name>ranger.web.authentication.alt-kerberos.algorithm</name>
+		<value>SHA256withRSA</value>
+	</property>
+	<property>
+		<name>ranger.web.authentication.alt-kerberos.certificatedir</name>
+		<value>/etc/verticloud/certs/sso</value>
```

Comment:
same as with auth_secret - let's use something generic here and overwrite in chef.